### ISSUE TITLE
[BUU] Specs for bulk products feature parity

### ISSUE THREAD

**Issue by dacook** [openfoodfoundation/openfoodnetwork#11845]
We have [tests on the old bulk edit products screen](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/admin/bulk_product_update_spec.rb), which we should make sure is covered by the [new bulk edit products spec](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/admin/products_v3/products_spec.rb)

This would be necessary before removing the `admin_style_v3` feature toggle.

```
As an Administrator
  I want to be able to manage products in bulk
  listing products
    displays a list of products
    displays a message when number of products is zero
    displays a select box for suppliers, with the appropriate supplier selected
    displays an on hand count in a span for each product
    displays 'on demand' for any variant that is available on demand
    displays a select box for the unit of measure for the product's variants
    displays a text field for the item name when unit is set to 'Items'
```

**Comment by filipefurtad0** [comment]
I think there are some test cases here which are blocked by https://github.com/openfoodfoundation/openfoodnetwork/issues/11059, because they concern some functionalities which do not yet exist.

I've updated the tests which I think do not relate to future features (although they might break once the page is changed again).

I think it's generally a good idea to have the branch green, so I'll signal these pending test cases, and mark this issue as blocked.

**Comment by dacook** [comment]
I believe this is now unblocked, because all the items listed above are now implemented. 
Hopefully, most of the checks in `bulk_product_update_spec.rb` are already covered in `products_v3/products_spec.rb`, but it will be worth reviewing to make sure. If there are any gaps, we can add them to the new spec file.

Then when we disable the old products screen, we can remove the old spec file at the same time.

**Comment by filipefurtad0** [comment]
Thanks for signalling @dacook :raised_hands:

**Comment by mkllnk** [comment]
@filipefurtad0, when do you think this issue will be done?

**Comment by filipefurtad0** [comment]
Hi @mkllnk , 
Depending on what's missing, I think we should be done during next week.

**Comment by dacook** [comment]
🎉
